packages/web/src/pages/chat-page/components/ChatMessagePlaylist.tsx

```diff
@@ -0,0 +1,29 @@
+import { Collection, Status } from '@audius/common'
+
+import MobilePlaylistTile from 'components/track/mobile/ConnectedPlaylistTile'
+
+type ChatMessagePlaylistProps = {
+  playlist: Collection | undefined | null
+  status: Status
+  errorMessage: string | null | undefined
+}
+
+export const ChatMessagePlaylist = ({
+  playlist,
+  status,
+  errorMessage
+}: ChatMessagePlaylistProps) => {
+  if (status === Status.ERROR) {
+    return (
+      // todo
+      <div>error</div>
+    )
+  }
+  // - numLoadingSkeletonRows
+  return (
+    <div>yolo</div>
```

Comment:
Implementing in upcoming PR to keep this one small

Reply:
just in case, let's make this return null instead of "yolo" 😂